### ISSUE TITLE
Reports facebook verification unreachable

### ISSUE THREAD

**Issue by derekmeans** [keybase/client#5618]
Keybase GUI Version: 1.0.18-20170130165558+0fcd377
OSX: 10.11.6

Website version reports facebook as verified.  OSX GUI version does not.

**Comment by cjb** [comment]
@oconnor663 any ideas?

**Comment by oconnor663** [comment]
Hmm, I'm able to `keybase id derekmeans` just fine. Is that the right user to be looking at? Here's what I see:

![image](https://cloud.githubusercontent.com/assets/860932/22484973/c4be1944-e7d1-11e6-98b6-5e8d9050dff6.png)

**Comment by derekmeans** [comment]
Yea that's me.  For some reason only the osx gui app claims it can't see my
Facebook verification key

**Comment by oconnor663** [comment]
Any chance you're on a network that disallows Facebook? Maybe try to load your profile in the browser? I'm also curious what happens if you run `keybase id --skip-proof-cache` in the terminal.

**Comment by derekmeans** [comment]
You are right it looks to be something with the way the network I was on is
configured.  I'm now on my home network and that's reporting with no issues.

Derek Means
http://www.derekmeans.com
derek.r.means@gmail.com
<https://mail.google.com/mail/u/0/?view=cm&fs=1&tf=1&to=derek.r.means@gmail.com>
917-725-1272

**Comment by bruceburge** [comment]
Though, not related to derek's issue, the same problem happens if you turn off the facebook application platform. 

I think requiring users to use, and leave active the facebook applications platform is highly undesirable, It leaves user open to all those application spam messages, and expooses a lot of your information to applications used by friends.

**Comment by oconnor663** [comment]
@bruceburge could you give some more detail? At what point in the proving/verifying process does that setting break things?

**Comment by bruceburge** [comment]
@oconnor663 After reproducing the process, the issue comes up, that if you disable the platform, and purge platform content, the purging of the content removes the post made by keybase, which cause the proof to fail.

The proof is failing within an hour of removing the post, what is the polling frequency of proof checks, as an hour seems a bit high.

**Comment by oconnor663** [comment]
Good to know. I don't know if we're going to be able to solve this one, without hurting the typical experience too much. The platform-based way we have you create the post is actually doing two things for us:

1. It lets us assemble the post (attachment) for the user without needing them to copy-paste stuff into Facebook. This is very nice to have, but not strictly necessary, and we do without it on Twitter for example.
2. It lets us _discover_ the post URL. Facebook prevents us from scraping your posts, even public ones, without going through the full Facebook Login flow. The only opportunity they give us to learn your post ID without needing that login flow is in a callback that we get from that platform-based post link. If we instead have you paste in post content manually, Twitter style, we'll _also_ need you to give us back a permalink to the post you just made, which turns into a pretty awful experience.

**Comment by bruceburge** [comment]
I agree, all other alternatives don't seem great, I suppose with all
things, there has to be compromise.

The only thing I can think that might help the situation, is perhaps a
reminder that if you previously had the platform turned off, you should
disable it again, but not remove the post. mayhaps a link to (
https://www.facebook.com/settings?tab=applications) but I don't know how
often that link changes, or you guys want to get into the steeple chase of
keeping up with facebooks url/link changes.

Thanks for taking the time to explain this.

**Comment by agentbond007** [comment]
Is it possible that some network providers are blocking Keybase login access?  It seems when using some VPN providers and Starbucks that I get a consistent message " Keybase is currently unreachable ..Trying to reconnect you. "     Is there a list of Auth servers that is runs through? 
<img width="511" alt="screen shot 2017-02-09 at 1 13 13 pm" src="https://cloud.githubusercontent.com/assets/3475353/22809062/8d7f9380-eee4-11e6-9cf3-b51ed3dc71ec.png">

**Comment by agentbond007** [comment]
▶ ERROR API network error: doRetry failed, attempts: 6, timeout 11.390625s, last err: Get https://api.keybase.io/_/api/1.0/sesscheck.json: dial tcp 54.84.82.15:443: connect: operation not permitted (error 1601)

**Comment by agentbond007** [comment]
<img width="643" alt="screen shot 2017-02-09 at 4 31 43 pm" src="https://cloud.githubusercontent.com/assets/3475353/22809178/4f95389e-eee5-11e6-83fe-1890c17974a2.png">

**Comment by maxtaco** [comment]
The last error is because our client comes with its own CA (no reason to trust verisign) that's not in your browser.  Try removing the `api.` and the above call should work

Though I don't understand the problem with your network connectivity otherwise

**Comment by agentbond007** [comment]
Thanks @maxtaco . Figured it out. I have a subnet specific firewall rules and when I'm on public networks, I have much tighter rules. Keybase was being block on certain ports.  Fixed and working. Appreciate fast response.

**Comment by diego898** [comment]
I was having a similar problem, where I block twitter through my hosts file and the osx client was showing:

![image](https://cloud.githubusercontent.com/assets/742407/25969814/1c1f4a60-364b-11e7-9a32-9f5f8c94ae41.png)

though the website was showing I was verified. I then ublocked it, and ran `keybase id diego` and it came up fine. I then told keybase "I fixed it -recheck" but it is still showing up as red. Restarting the OSX client did not fix it.
